src/main/java/com/displee/editor/controller/MainController.kt

```diff
@@ -348,13 +429,18 @@ class MainController : Initializable {
 				try {
 					CS2Reader.readCS2ScriptNewFormat(data, id, config.unscrambled, config.disableSwitches, config.disableLongs)
 				} catch(e: Throwable) {
-					error++
-					if (error >= 2) {
-						break
+					if (config.version == 179) {
+						println(e)
+						println("id $id")
+
 					}
+//					error++
+//					if (error >= 2) {
```

Comment:
Why exactly is this commented?

Reply:
I can't remember, I think I was debugging to see which new CS2s were erroring. Can probably throw it back in, it's been a while since I did this.

Reply:
just read my code, was to get all scripts that were broken, can add back in for production until next time an upgrade is needed